src/sms/adapters/inbound/fastapi_/routes.py

```diff
@@ -38,6 +37,30 @@
     description="The database and collection to query. Format: db_name.collection",
 )
 
+query_params_desc = (
+    "Query parameters used to specify the documents affected by the request."
+)
+query_params_open_api: dict[str, Any] = {
+    "parameters": [
+        {
+            "description": query_params_desc,
+            "in": "query",
+            "name": "Criteria",
+            "required": False,
+            "schema": {
+                "description": query_params_desc,
+                "title": "Criteria",
+                "type": "object",
+                "examples": [
+                    '{"name": "John"}',
+                    '{"_id": {"$eq": 1}}',
```

Comment:
```suggestion
                    '{"_id": {"$ne": "507f1f77bcf86cd799439011"}}',
```

since the `_id` is usually an objectId or a string and since `$eq` is normally not needed